AutomatedTesting/src/test/resources/features/DeletePlanetWithMoon.feature

```diff
@@ -0,0 +1,13 @@
+Scenario outline: Delete Planet from Planetarium Does Planet Have Moons Test Case
```

Comment:
Feature Files start with "Feature: ..." on the first line and "Scenario Outline: ..." on the following line indented.